src/management/labels/label-creator.html

```diff
@@ -1,6 +1,7 @@
 <h1 class="title">Label toevoegen</h1>
 
 <form name="creator.form" class="css-form" novalidate>
+    <div class="label-create-wrapper">
     <div class="row">
```

Comment:
Would maybe just indent the children so it's easier to read, or run Prettier manually on that one file perhaps to save time